config/update/thunder_update_8134.yml

```diff
@@ -0,0 +1,218 @@
+          placeholder: ''
+          size: 60
+        type: entity_reference_autocomplete
+  update_actions:
+    add:
+      content:
+        uid:
+          settings:
+            autocomplete: true
+    change:
+      content:
+        uid:
+          settings:
+            width: 100%
+          type: select2_entity_reference
+    delete:
+      content:
+        uid:
+          settings:
+            placeholder: ''
+            size: 60
+core.entity_form_display.media.instagram.default:
+  expected_config:
+    content:
+      uid:
+        settings:
+          placeholder: ''
+          size: 60
+        type: entity_reference_autocomplete
+  update_actions:
+    add:
+      content:
+        uid:
+          settings:
+            autocomplete: true
+    change:
+      content:
+        uid:
+          settings:
+            width: 100%
+          type: select2_entity_reference
+    delete:
+      content:
+        uid:
+          settings:
+            placeholder: ''
+            size: 60
+core.entity_form_display.media.pinterest.default:
+  expected_config:
+    content:
+      uid:
+        settings:
+          placeholder: ''
+          size: 60
+        type: entity_reference_autocomplete
+  update_actions:
+    add:
+      content:
+        uid:
+          settings:
+            autocomplete: true
+    change:
+      content:
+        uid:
+          settings:
+            width: 100%
+          type: select2_entity_reference
+    delete:
+      content:
+        uid:
+          settings:
+            placeholder: ''
+            size: 60
+core.entity_form_display.media.twitter.default:
+  expected_config:
+    content:
+      uid:
+        settings:
+          placeholder: ''
+          size: 60
+        type: entity_reference_autocomplete
+  update_actions:
+    add:
+      content:
+        uid:
+          settings:
+            autocomplete: true
+    change:
+      content:
+        uid:
+          settings:
+            width: 100%
+          type: select2_entity_reference
+    delete:
+      content:
+        uid:
+          settings:
+            placeholder: ''
+            size: 60
+core.entity_form_display.media.video.default:
+  expected_config:
+    content:
+      uid:
+        settings:
+          placeholder: ''
+          size: 60
+        type: entity_reference_autocomplete
+  update_actions:
+    add:
+      content:
+        uid:
+          settings:
+            autocomplete: true
+    change:
+      content:
+        uid:
+          settings:
+            width: 100%
+          type: select2_entity_reference
+    delete:
+      content:
+        uid:
+          settings:
+            placeholder: ''
+            size: 60
+core.entity_form_display.node.article.default:
+  expected_config:
+    content:
+      uid:
+        settings:
+          placeholder: ''
+          size: 60
+        type: entity_reference_autocomplete
+  update_actions:
+    add:
+      content:
+        field_paragraphs:
```

Comment:
I would exclude these additional fields (`field_paragraphs`, `field_tags`, `field_teaser_media`) from this update.
Problem is if someone has already configured something we will overwrite that. And update has stated something different - `Use select2 widget for author field.`.

I would rather make new updates dedicated to different configuration changes. If we take paragraphs `closed_mode_threshold` as an example. We have that configuration change for taxonomy terms too. 😞